web_src/less/helpers.less

```diff
@@ -1,6 +1,7 @@
 .gt-df { display: flex !important; }
 .gt-di { display: inline !important; }
 .gt-dif { display: inline-flex !important; }
+.gt-db { display: block !important; }
```

Comment:
What do you think about this proposal:
* https://github.com/go-gitea/gitea/issues/23011

Reply:
I think HesterG's answer is definitely yes because she already gave your proposal a thumbs up. 😄

But maybe it's a good idea to keep the old style before refactoring?

Reply:
> What do you think about this proposal:
> 
> * [[Proposal] Use tailwind style CSS class names for legacy helpers #23011](https://github.com/go-gitea/gitea/issues/23011)

I pretty agreed with [this comment](https://github.com/go-gitea/gitea/issues/23011#issuecomment-1438538003),  but i am note sure if we could partially introduce tailwind, like only introduce widely used css like we now have inside `helper.less`